src/games/strategy/net/ServerMessenger.java

```diff
@@ -52,7 +52,7 @@
   private final CopyOnWriteArrayList<IMessageListener> listeners = new CopyOnWriteArrayList<>();
   private final CopyOnWriteArrayList<IMessengerErrorListener> errorListeners =
       new CopyOnWriteArrayList<>();
-  private final CopyOnWriteArrayList<IConnectionChangeListener> connectionListeners =
+  protected final CopyOnWriteArrayList<IConnectionChangeListener> connectionListeners =
```

Comment:
Quick reminder for me.. I need to revert this change since this is a result of testing wich didn't worked